src/main/resources/messages/messages.properties

```diff
@@ -663,6 +673,7 @@ message.download_translations.preserve_hierarchy_warning=Because the 'preserve_h
 message.language.list=@|yellow %s|@ @|green %s|@
 
 message.branch.list=@|yellow #%d|@ @|green %s|@
+message.branch.clone=@|yellow #%s|@ @|green %s|@
```

Comment:
Why not use the `message.branch.list' message instead of creating this new one? It accepts the same parameters